Added files of fire detection using Image Processing

Rajeswari0410 opened the pull request:
Fixed Issue #301 
I have added a folder named Fire Detection using Image Processing which contains two programs(CNN) for Fire-detection using Image Processing. Main1 is having an accuracy of 93% and Main2 have an accuracy of >95%. Thank you for alloting it to me.

Commit:
Added files of fire detection

gpriya32 commented:
Pls, add a dataset for this. Also in the main2 notebook, there is no accuracy plot.

gpriya32 commented:
Also, save the model in h5 format and add it to the folder

ashishsahu1 commented:
@Rajeswari0410 Updates?

Rajeswari0410 commented:
Hlo,
      There is a problem in adding datasets. So, I couldn't add them in the
folder

On Wed, May 12, 2021, 16:01 Ashish Sahu ***@***.***> wrote:

> @Rajeswari0410 <https://github.com/Rajeswari0410> Updates?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ashishsahu1/ML-ProjectYard/pull/350#issuecomment-839663556>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARWDX7WWJPTF6VMWE6L2Y5TTNJKJXANCNFSM43H7LS6A>
> .
>

gpriya32 commented:
Provide link to your dataset in readme then

Rajeswari0410 commented:
Ok...I will provide drive link in Readme.

On Sat, May 15, 2021, 12:17 Priyanka Gupta ***@***.***> wrote:

> Provide link to your dataset in readme then
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ashishsahu1/ML-ProjectYard/pull/350#issuecomment-841610928>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARWDX7Q3KGRTOQCILQP6AQDTNYKGVANCNFSM43H7LS6A>
> .
>

Commit:
Modified

Commit:
Modified files

Rajeswari0410 commented:
Hi! thanks for remainding me, there was a problem with datasets uploading. I think it is due to large size. Similarly I am facing trouble uploading it the model in h5 format. I have changed only for 1 method. I have added the datasets links in README file.
Later I will sort out the problem and add the second method too. I think the problem is because of large size. You can merge this PR. Thank you once again.
